### ISSUE TITLE
gmt2kml appears to fail when using -Ncol

### ISSUE THREAD

**Issue by MarceloBanik** [GenericMappingTools/gmt#1037]
**Description of the problem**
gmt2kml -Nt is working fine, thanks for taking a look at it.

I was trying to figure out how to use the option -Ncol, and appears to have an inconsistency, plus my poor understanding of how to use it.

I expected the following commands should be equivalent:
```
echo "-45 -30 sjsp" | gmt 2kml -Fs -Nt
echo "-45 -30 sjsp" | gmt 2kml -Fs -N2
```
But the second command returns:
```
gmt2kml [ERROR]: Data file has fewer columns than implied by -N<col>
```
Including option -i, I couldn't avoid it to convert the string "sjsp" to "NaN", but at least it get the column text as I expected:
```
echo "-45 -30 sjsp" | gmt 2kml -i0-2 -Fs -N2 | grep name
                        <name>stdin</name>
                                <name>Point Set 0</name>
                                        <name>NaN</name>
```
Looking in the "man gmt", option -i, it says:
“Normally,  any  trailing text is read but when i is used you must add the column t to retain the text”.
But I dind’t get how to retain the text.

I noted that option -N could get the text from second column:
```
echo "-45 -30 sjsp" | gmt 2kml -Fs -N1 | grep name
                        <name>stdin</name>
                                <name>Point Set 0</name>
                                        <name>-30</name>
```
But, as shown before, it couldn't get the next column. What appears odd to me.

**System information**

* Operating system: Ubuntu 18.04.2 LTS
* Version of GMT: 6.1.0_5a9a310_2019.06.29

**Comment by PaulWessel** [comment]
I do not think there are any bugs here.  There are numerical columns and trailing text.  **-N** refers to numerical columns.  Your **-N**2 gives an error since there is only two numerical columns and **-N**2 refers to a rising 3rd column.

`echo "-45 -30 sjsp" | gmt convert -i0-1,t
`

will print out the input record, including the trailing text.  Leave off the **t** and you just bet the numerical columns you selected.

if you want to use **-N** to get the trialing text, the man pages says use **-Nt**.

**Comment by MarceloBanik** [comment]
Ok. Probably I misunderstanding how to use gmt2kml -Ncol.
My real problem is that I have a record like:
-45:51:42.25600 -23:12:25.67670 SJSP SP São José dos Campos
and I want to use only the 3rd colums as a label. Then I could solve it using:
```
echo "-45:51:42.25600 -23:12:25.67670 SJSP SP São José dos Campos" | awk '{print $1,$2,$3}' | gmt 2kml -Fs -Nt
```
Then I misunderstanding the usage of option -Ncol as a way to choose a column with the text that I wish, avoiding the use of awk.
Probably here is the wrong place to ask how to use it properly, but let me show a last attempt to use it:
```
echo "-45:51:42.25600 -23:12:25.67670 90" | gmt 2kml -Fs -N2 | grep 90
                                        <name>90</name>
```
Then now I could get a label "90" in the third column, but it couldn't be "91":
```
echo "-45:51:42.25600 -23:12:25.67670 91" | gmt 2kml -Fs -N2 > /dev/null
gmt2kml [ERROR]: Column selected for latitude-formatting has values that exceed +/- 90; set to NaN
```
Probably it just show that I didn't get how to use -Ncol. But is a little odd how it works in these cases.

**Comment by PaulWessel** [comment]
Now that looks like a bug (imposing latitude restriction on column 3).  Will check that.

**Comment by PaulWessel** [comment]
BTW, you could consider opening a feature request to let the trailing text column flag _t_ take modifiers that allow selection of words.  E.g., **-i**1-3,_t_:2  would consider the trailing text string as a list of words and select the third word instead of the whole thing.  We do something like that in movie **-L**.

**Comment by PaulWessel** [comment]
This latitude-formatting issue is now fixed in master and 6.0 branches.

**Comment by seisman** [comment]
Closed by #1048.
